recipes/config.rb

```diff
@@ -7,30 +7,31 @@
 
 include_recipe 'stackstorm::user'
 
+# Ensure directory for config is available
+directory ':create /etc/st2 directory' do
+  path '/etc/st2'
+  owner 'root'
+  group 'root'
+  mode '0755'
+  action :create
+end
```

Comment:
cc @shortdudey123, in context with https://github.com/StackStorm/chef-stackstorm/pull/42#discussion_r88315953.
Since we really touch `/etc/st2/st2.conf`, ensuring here that directory `/etc/st2` is manageable.